packages/algorand-js/src/interpreter/opcode.ts

```diff
@@ -98,13 +98,13 @@ export class Op {
   getAccount (accountIndex: bigint, interpreter: Interpreter): AccountState {
```

Comment:
this function should be in in runtime, or be a method of `state` object.

Reply:
okay, moved to runtime